ramalama/common.py

```diff
@@ -46,6 +46,14 @@ def get_engine():
             apple_vm(engine)
         return engine
 
+    if (
```

Comment:
This will break the use case of running RamaLama with podman-remote, which Podman Desktop uses for running RamaLama on Windows.

Reply:
Just check for /run/.toolboxenv

Reply:
Done